Source/Yoshis Woolly World/rules.txt

```diff
@@ -0,0 +1,196 @@
+[Definition]
```

Comment:
You can just replace the Woolly World pack I pushed to the repo an hour ago.

Reply:
All righty, done